storybook/src/components/Spotlight/Spotlight.stories.tsx

```diff
@@ -17,7 +17,7 @@ const meta = {
     <Spotlight as={as} color={color}>
       <Grid paddingVertical="medium">
         <Grid.Cell span="all">
-          <Blockquote color={!color || !['green', 'yellow'].includes(color) ? 'inverse' : undefined}>
+          <Blockquote color={!color || ['blue', 'green', 'magenta'].includes(color) ? 'inverse' : undefined}>
```

Comment:
`purple` should be in this list I think. And in other components (Avatar and Badge) we use a black text color for `blue` backgrounds. Do we want to diverge here?

Reply:
Purple is the default, so `!color` catches that.

I removed blue (I think you meant azure) that we don’t use for Spotlight.